safe/utilities/keyword_io.py

```diff
@@ -457,35 +463,127 @@ def _keyword_to_row(self, keyword, value, wrap_slash=False):
         return row
 
     @staticmethod
-    def _threshold_to_row(keyword_value):
+    def _threshold_to_row(thresholds_keyword):
         """Helper to make a message row from a threshold
 
         We are expecting something like this:
 
-            "{'high': [0, 3], "
-            "'medium': [3, 5], "
-            "'low': [5, 10]}"
+        {
+            'thresholds': {
+                'structure': {
+                    'ina_structure_flood_hazard_classification': {
+                        'classes': {
+                            'low': [1, 2],
+                            'medium': [3, 4],
+                            'high': [5, 6]
+                        },
+                        'active': True
+                    },
+                    'ina_structure_flood_hazard_4_class_classification':
+                    {
+                        'classes': {
+                            'low': [1, 2],
+                            'medium': [3, 4],
+                            'high': [5, 6],
+                            'very_high': [7, 8]
+                        },
+                        'active': False
+
+                    }
+                },
+                'population': {
+                    'ina_population_flood_hazard_classification': {
+                        'classes': {
+                            'low': [1, 2.5],
+                            'medium': [2.5, 4.5],
+                            'high': [4.5, 6]
+                        },
+                        'active': False
+                    },
+                    'ina_population_flood_hazard_4_class_classification':
+                    {
+                        'classes': {
+                            'low': [1, 2.5],
+                            'medium': [2.5, 4],
+                            'high': [4, 6],
+                            'very_high': [6, 8]
+                        },
+                        'active': True
+                    }
+                },
+            },
 
         Each value is a list with exactly two element [a, b], where a <= b.
 
-        :param keyword_value: Value of the keyword to be rendered. This must
-            be a string representation of a dict, or a dict.
-        :type keyword_value: basestring, dict
+        :param thresholds_keyword: Value of the keyword to be rendered. This
+            must be a string representation of a dict, or a dict.
+        :type thresholds_keyword: basestring, dict
 
         :returns: A table to be added into a cell in the keywords table.
         :rtype: safe.messaging.items.table
         """
-        if isinstance(keyword_value, basestring):
-            keyword_value = literal_eval(keyword_value)
+        if isinstance(thresholds_keyword, basestring):
+            thresholds_keyword = literal_eval(thresholds_keyword)
+
+        table = m.Table(style_class='table table-condensed table-striped')
+
+        i = 0
```

Comment:
Ahah, it's a long time in python I didn't see that. I usually use `enumerate` ;-)
No worries. We can keep it, I guess it will not fit one your line.